src/main/resources/swagger.yaml

```diff
@@ -0,0 +1,949 @@
+          schema:
+            type : string
+        - name: amr_values
+          in: query
+          required: false
+          description: AMR Values.
+          schema:
+            type : string
+        - name: request
+          in: query
+          required: false
+          description: This parameter enables OpenID Connect requests to be passed in a single, self-contained parameter and to be optionally signed and/or encrypted. The parameter value is a Request Object value. It represents the request as a JWT whose Claims are the request parameters.
+          schema:
+            type : string
+        - name: request_uri
+          in: query
+          required: false
+          description: This parameter enables OpenID Connect requests to be passed by reference, rather than by value. The request_uri value is a URL using the https scheme referencing a resource containing a Request Object value, which is a JWT containing the request parameters.
+          schema:
+            type : string
+        - name: request_session_id
+          in: query
+          required: false
+          description: Request session id.
+          schema:
+            type : string
+        - name: session_id
+          in: query
+          required: false
+          description: Session id of this call.
+          schema:
+            type : string
+        - name: origin_headers
+          in: query
+          required: false
+          description: Origin headers. Used in custom workflows.
+          schema:
+            type : string
+        - name: code_challenge
+          in: query
+          required: false
+          description: PKCE code challenge.
+          schema:
+            type : string
+        - name: code_challenge_method
+          in: query
+          required: false
+          description: PKCE code challenge method.
+          schema:
+            type : string
+        - name: custom_response_headers
+          in: query
+          required: false
+          description: Custom Response Headers.
+          schema:
+            type : string
+        - name: claims
+          in: query
+          required: false
+          description: Requested Claims.
+          schema:
+            type : string
+        - name: auth_req_id
+          in: query
+          required: false
+          description: CIBA authentication request Id.
+          schema:
+            type : string
+      responses:
+        200:
+          description: OK
+          content: {}
+        302:
+          description: Invalid parameters are provided to endpoint. 
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ErrorResponse'   
+  /authorize(post):
```

Comment:
there is not `/authorize(post)` path. Instead we should have just `post` declaration above for `/authorize`.

Reply:
Rectified